src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -351,7 +351,9 @@
 
     <Exec Command="$(FullCommand)"
           WorkingDirectory="$(ProjectDirectory)"
-          EnvironmentVariables="@(EnvironmentVariables)" />
+          EnvironmentVariables="@(EnvironmentVariables)">
```

Comment:
I wonder if the this exec (orchestrator to the repo outer build) should also set `IgnoreStandardErrorWarningFormat`? https://github.com/dotnet/arcade/blob/9aa3f2e68b30ac51823dd444e8cb962e058c5699/src/Microsoft.DotNet.Arcade.Sdk/tools/SourceBuild/SourceBuildArcadeBuild.targets#L217C7-L217C39

@mmitche

Reply:
Probably. I think most of the time in CI we're not spitting log info to the console so we're not hitting error detection.